public/app/core/services/backend_srv.ts

```diff
@@ -539,7 +539,9 @@ export class BackendSrv implements BackendService {
       queryParams.set('accesscontrol', 'true');
     }
 
-    return this.get<FolderDTO>(`/api/folders/${uid}?${queryParams.toString()}`);
+    return this.get<FolderDTO>(`/api/folders/${uid}?${queryParams.toString()}`, undefined, undefined, {
+      showErrorAlert: false,
+    });
```

Comment:
this bit concerns me slightly, because it's happening for _every_ call to `getFolderByUid` not just the call from `initDashboard`. i've ran it locally and if we go to a folder with the wrong uid we still get a `folder not found` error toast being shown, but i'm not sure if there are other scenarios i'm forgetting where we should show an error toast if the folder doesn't exist. can we parameterise it in any way?

Reply:
Valid concern, and I'm happy to do this differently if we want. I don't feel super strongly about this.

My rationale is that there shouldn't be anywhere in Grafana that relies on this toast to be the indicator that something failed. The page itself should show the error in context, rather than relying on the temporary toast.

(I hate these automate toasts, in case you can't tell 😉)